x-pack/plugins/security_solution/server/lib/detection_engine/signals/send_telemetry_events.test.ts

```diff
@@ -58,13 +64,20 @@ describe('sendAlertTelemetry', () => {
               '@timestamp': 'x',
               key3: 'hello',
               data_stream: {},
+              event: {
+                id: 'baz',
+              },
```

Comment:
Can you also add 2 more cases where `event.id` doesn't exist and another for when there is no key in the join map correlating to the alert id?

Reply:
Sure, can do.